packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -30,125 +42,90 @@ export default {
-    });
-  }
-};
 
-/**
- *  The different variants of the brandshape.
- */
-export const Variants = {
-  name: 'Variants',
-  parameters: { controls: { exclude: ['variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'variant'
-        }
-      },
-      options: {
-        templateBackgrounds: {
-          alternate: 'y',
-          colors: ['white', 'white', 'rgb(var(--sd-color-primary, 0 53 142))']
-        }
-      },
-      args,
-      constants: increaseColumnWidth()
-    });
-  }
-};
+export const Variant = {
+  name: 'Variant',
+  render: () => html`
+    <div class="mb-5">
+      <sd-brandshape variant="primary">
+        <p class="sd-headline sd-headline--lg text-white">Primary Variant Example</p>
+      </sd-brandshape>
+    </div>
 
-/**
- * The different breakpoints of the brandshape.
- */
-export const Breakpoints = {
-  name: 'Breakpoints',
-  render: (args: any) => {
-    const getSize = (breakpoint: string): string => {
-      const breakpoints: { [key: string]: string } = {
-        '≤ 414px': '414px',
-        '> 414px': '415px',
-        '> 640px': '641px'
-      };
-      return breakpoints[breakpoint] || '100%';
-    };
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'query width = ...',
-          values: ['≤ 414px', '> 414px', '> 640px'].map(breakpoint => {
-            return {
-              title: breakpoint,
-              value: `<div style="width: ${getSize(breakpoint)}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      args
-    });
-  }
+    <div class="mb-5">
+      <sd-brandshape variant="neutral-100">
+        <p class="sd-headline sd-headline--lg">Neutral-100 Variant Example</p>
+      </sd-brandshape>
+    </div>
+
+    <div class="bg-primary">
+      <sd-brandshape variant="white">
+        <p class="sd-headline sd-headline--lg">White Variant Example</p>
+      </sd-brandshape>
+    </div>
+
+    <div>
+      <sd-brandshape variant="border-primary">
+        <p class="sd-headline sd-headline--lg">Border Primary Variant Example</p>
+      </sd-brandshape>
+    </div>
+
+    <div class="bg-primary">
+      <sd-brandshape variant="border-white">
+        <p class="sd-headline sd-headline--lg text-white">Border White Variant Example</p>
+      </sd-brandshape>
+    </div>
+
+    <div>
+      <sd-brandshape variant="image">
+        <p class="sd-headline sd-headline--lg">Image Variant Example</p>
+        <img slot="image" style="transform:translateY(-30%);" src="./placeholders/images/generic.jpg" alt="Generic" />
+      </sd-brandshape>
+    </div>
+  `
 };
 
 /**
- * Use the `base`, `content`, `shape-top`, `shape-middle` or `shape-bottom` part selectors to customize the brandshape.
+ * By default, all shapes are displayed but it's possible to display only specific shapes by utilizing the `shapes` attribute.
```

Comment:
Code-only: Use the “shapes” attribute to only display specific parts of the brandshape.

Figma-only: Hide layers to only display specific parts of the brandshape.

Reply:
Updated